test/elm/interval/test.coffee

```diff
@@ -2124,3 +2167,159 @@ describe 'DecimalIntervalExpand', ->
     # define BadPerMinute: expand { Interval(2.1, 4.1] } per 0.5 minute
     a = @badPerMinute.exec(@ctx)
     should.not.exist(a)
+
+describe 'SameAs', ->
+  @beforeEach ->
+    setup @, data
+
+  it 'returns null when both intervals values are null', ->
+    # define NullBoth: Interval[null,null] same as Interval[null,null]
+    a = @nullBoth.exec(@ctx)
+    should.not.exist(a)
```

Comment:
It is better to use `should(a).be.null()` to test for null, as should.not.exist will pass for `undefined` as well.